tests/end-to-end/simple/App.cs

```diff
@@ -0,0 +1,203 @@
+                },
+                {
+                    "False", new Dictionary<string, bool>
+                    {
+                        { "false", false },
+                    }
+                },
+            };
+            var lists = new Dictionary<string, Dictionary<string,string[]>> 
+            {
+                {
+                    "Candidates", new Dictionary<string, string[]>
+                    {
+                        {
+                            "Empty", new string[] 
+                            {
+                            }
+                        },
+
+                        {
+                            "Singleton", new string[] 
+                            {
+                                "One",
+                            }
+                        },
+
+                        {
+                            "Pair", new string[] 
+                            {
+                                "One",
+                                "Two",
+                            }
+                        },
+
+                    }
+                },
+            };
+            var numbers = new Dictionary<string, Dictionary<string,int>> 
+            {
+                {
+                    "Prime", new Dictionary<string, int>
+                    {
+                        { "Eleven", 11 },
+                        { "Thirteen", 13 },
+                        { "Seventeen", 17 },
+                    }
+                },
+            };
+            var strings = new Dictionary<string, Dictionary<string,string>> 
+            {
+                {
+                    "Foos", new Dictionary<string, string>
+                    {
+                        { "Foo1", "Foo1" },
+                        { "Foo2", "Foo2" },
+                    }
+                },
+                {
+                    "Bars", new Dictionary<string, string>
+                    {
+                        { "Bar", "Bar" },
+                    }
+                },
+            };
+            var table = new Dictionary<string, Dictionary<string,object>> 
+            {
+                {
+                    "Values", new Dictionary<string, object>
+                    {
+                        { "Boolean", true },
+                        { "Float", 3.14 },
+                        {
+                            "List", new string[] 
+                            {
+                                "1",
+                                "2",
+                                "3",
+                            }
+                        },
+
+                        { "Number", 42 },
+                        { "String", "Baz" },
+                    }
+                },
+            };
+            bool isUs = Fn.Select(0, Fn.Split("-", Region)) == "us";
+            bool isUsEast1 = Region == "us-east-1";
+            bool isLargeRegion = isUsEast1;
+            var queue = new CfnQueue(this, "Queue", new CfnQueueProps
```

Comment:
Again I personally like the Gap + comment to label this as the resources section.